package/rspack/rules/sass.js

```
@@ -0,0 +1,22 @@
+    sassOptions: {
+      quietDeps: true
+    }
+  }
+}
+
+// getStyleRule handles css-loader and postcss-loader internally
+// We only need to pass the extraction loader and sass-loader
+module.exports = getStyleRule(/\.(scss|sass)$/i, [
+  inliningCss ? "style-loader" : CssExtractRspackPlugin.loader,
+  sassLoader
+])
```

ESLint

[error] Insert `⏎`

(prettier/prettier)